cmd/snap-confine/snap-confine.apparmor.in

```diff
@@ -313,6 +313,18 @@
     @{HOME}/ r,
     @{HOME}/snap/{,*/,*/*/} rw,
 
+    # Special case for *classic* snaps that are used by users with home dirs
+    # in /var/lib/. Like jenkins, postgresql, mysql, pupett, ...
+    # (see https://forum.snapcraft.io/t/9717)
+    # TODO: this can be removed once we support home-dirs outside of /home
+    #       better
+    /var/ r,
+    /var/lib/ r,
+    # These should both have 'owner' match but due to LP: #1466234, we can't
+    # yet
+    /var/lib/*/ r,
+    /var/lib/*/snap/{,*/,*/*/} rw,
```

Comment:
If @jdstrand agrees we might simplify this to just
```
/var/lib/*/snap/** rw,
```

Reply:
Please don't do the simplification @zyga suggested. We want snap-confine to be able to create `/var/lib/<user>/snap/<snap name>/<common, revision, current>`, but not anything else (ie, *not* the data in these directories).